flash_cards/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+# from django.utils.text import slugify
 
 # Create your models here.
+class FlashCard(models.Model):
+    title = models.CharField(max_length=50)
+    description = models.TextField(max_length=1000, null=True, blank=True)
+    deck = models.ForeignKey('Deck', on_delete=models.CASCADE)
+
```

Comment:
I think we may want to sort these by date in the future, so this may be a good add (we'll all need to migrate again after we pull if we do this)
```suggestion
created_at = models.DateTimeField(auto_now_add=True)

```